src/framework/preferences/html/PrefsEditorTemplate-lineSpace.html

```diff
@@ -1,5 +1,6 @@
 <div class="flc-prefsEditor-line-space">
     <h2><span class="fl-icon-line-space"></span><label class="flc-prefsEditor-line-space-label" for="line-space"></label></h2>
```

Comment:
If the `<label>` in the header of the panel are not actually being used as labels, could you change them to be `<span>`.